src/indexer/sett-snapshots-indexer.ts

```diff
@@ -78,19 +81,26 @@ function settToSnapshot(chainNetwork: ChainNetwork): (settToken: string) => Prom
   };
 }
 
+const captureSnapshot = async (network: ChainNetwork, sett: SettDefinition): Promise<CachedSettSnapshot | null> => {
+  try {
+    const snapshotTranslateFn = settToSnapshot(network);
+    // purposefully await to leverage try / catch
+    const result = await snapshotTranslateFn(sett.settToken);
```

Comment:
adding new setts who had no interactions yet was breaking the caching due to all or none nature of promise.all